CHANGELOG.md

```diff
@@ -90,7 +90,10 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 * (simulation) [#18196](https://github.com/cosmos/cosmos-sdk/pull/18196) Fix the problem of `validator set is empty after InitGenesis` in simulation test.
 * (baseapp) [#18551](https://github.com/cosmos/cosmos-sdk/pull/18551) Fix SelectTxForProposal the calculation method of tx bytes size is inconsistent with CometBFT
 * (abci): [#19200](https://github.com/cosmos/cosmos-sdk/pull/19200) Ensure that sdk side ve math matches cometbft
-
+* (server) [#18994](https://github.com/cosmos/cosmos-sdk/pull/18994) Update server context directly rather than a reference to a sub-object
+* (client/server) [#18994](https://github.com/cosmos/cosmos-sdk/pull/18994) If context is nil, set context to background when setting server or client context
```

Comment:
I don't think we need this one either.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/18994/commits/d025eb5999134adb2e45630f32ac8e3043838e09